pkg/userdata/helper/kubelet.go

```diff
@@ -19,6 +19,7 @@ package helper
 import (
 	"bytes"
 	"fmt"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Please move this after "text/template" with a blank line and preferably with import alias corev1

```
import (
	"bytes"
	"fmt"
	"net"
	"text/template"

	corev1 "k8s.io/api/core/v1"
)
```